demos/horizontal_text_detection/python/horizontal_text_detection.py

```diff
@@ -195,38 +204,40 @@ def grab_frame(cap):
 last_display_time = time.time()
 app_start_time = time.time()
 
-while(True):
-    if not threads[i].is_initialized():
-        threads[i].set_input(grab_frame(cap))
-        i = (i + 1) % args.num_threads
-        continue
+if grab_frame(cap) == None:
+    print(f"[ERROR] Check camera input...")
```

Comment:
Same here, why cant we just quit an app if grab_frame returns None? This way we can save one indention level